ui/pages/confirmations/components/alerts/multiple-alert-modal/multiple-alert-modal.test.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import configureMockStore from 'redux-mock-store';
+import { fireEvent } from '@testing-library/react';
+import { Severity } from '../../../../../helpers/constants/design-system';
+import { renderWithProvider } from '../../../../../../test/lib/render-helpers';
+import {
+  MultipleAlertModal,
+  MultipleAlertModalProps,
+} from './multiple-alert-modal';
+
+describe('MultipleAlertModal', () => {
+  const OWNER_ID_MOCK = '123';
+  const FROM_ALERT_KEY_MOCK = 'from';
+  const handleButtonClickMock = jest.fn();
+  const onCloseMock = jest.fn();
+  const alertsMock = [
+    {
+      key: FROM_ALERT_KEY_MOCK,
+      severity: Severity.Warning,
+      message: 'Alert 1',
+      reason: 'Reason 1',
+      alertDetails: ['Detail 1', 'Detail 2'],
+    },
+    { key: 'data', severity: Severity.Danger, message: 'Alert 2' },
+  ];
+
+  const STATE_MOCK = {
+    confirmAlerts: {
+      alerts: { [OWNER_ID_MOCK]: alertsMock },
+      confirmed: {
+        [OWNER_ID_MOCK]: { [FROM_ALERT_KEY_MOCK]: false, data: false },
+      },
+    },
+  };
+  const mockStore = configureMockStore([])(STATE_MOCK);
+
+  const defaultProps: MultipleAlertModalProps = {
+    ownerId: OWNER_ID_MOCK,
+    handleButtonClick: handleButtonClickMock,
+    alertKey: FROM_ALERT_KEY_MOCK,
+    onClose: onCloseMock,
+  };
+
+  it('renders the multiple alert modal', () => {
```

Comment:
Should we have a test to verify the original `onButtonClick` is fired from the last alert modal?

And another to verify that `Got it` loads the next alert?